src/types/markers.js

```diff
@@ -421,7 +427,10 @@ export type NetworkPayload = {|
   pri: number, // priority of the load; always included as it can change
   count?: number, // Total size of transfer, if any
   // See all possible values in tools/profiler/core/platform.cpp
-  status: 'STATUS_START' | 'STATUS_STOP' | 'STATUS_REDIRECT',
+  status: NetworkStatus,
+  // The 2 following properties are not present in Gecko versions before .
```

Comment:
It would be good to note the version of that Firefox as well here. (I know it's not landed yet, but probably it's going to be this release (91) right?)

Also I feel like "2 following" is not really great long term, since some people may add another field in the middle. It would be better to add individual comments to both fields I think.

Reply:
Ah ah, I totally intended to add a version number (hence the space...)

And good idea for your other feedback.

Reply:
(I hope it will be in 91)

Reply:
Cool, we can always update the comment if it ends up in a different release.